docs/guide/etl.rst

```diff
@@ -564,9 +564,10 @@ Extension
 ---------
```

Comment:
It could be a general "hey bro your markup is wrong" warning. Either way, it's fixed now.